nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/test/java/org/apache/nifi/processors/aws/kinesis/stream/TestConsumeKinesisStream.java

```diff
@@ -117,6 +118,7 @@ public void testInvalidProperties() {
                         "'%s' validated against 'not-a-long' is invalid because Must be of format <duration> <TimeUnit> where <duration> is a non-negative integer and " +
                         "TimeUnit is a supported Time Unit, such as: nanos, millis, secs, mins, hrs, days\n" +
                         "'%s' validated against 'not-a-boolean' is invalid because Given value not found in allowed set 'true, false'\n" +
+                        "'Output Strategy' validated against 'not-an-enum-match' is invalid because Given value not found in allowed set 'USE_VALUE, USE_WRAPPER'\n" +
```

Comment:
I recommend removing this test case since evaluating error messages is generally not useful.